### PR TITLE
[macos] Add WKUIDelegate runOpenPanelWith bindings

### PR DESCRIPTION
**Pull request by chamons**
- https://bugzilla.xamarin.com/show_bug.cgi?id=58758

### CODE REVIEW

File: src/wkwebkit.cs [hunk 1]
```diff
@@ -297,6 +297,14 @@ interface WKWebsiteDataStore : NSCoding {
 		[Async]
 		void RemoveDataOfTypes (NSSet<NSString> websiteDataTypes, NSDate date, Action completionHandler);
 	}
+
+	[Mac (10,12)]
```
Comment (chamons) [inline src/wkwebkit.cs]: The whole type is marked with:

[Mac (10,10, onlyOn64 : true)]

but no harm in double marking the 64-bit only ness.

File: src/wkwebkit.cs [hunk 2]
```diff
@@ -297,6 +297,14 @@ interface WKWebsiteDataStore : NSCoding {
 		[Async]
 		void RemoveDataOfTypes (NSSet<NSString> websiteDataTypes, NSDate date, Action completionHandler);
 	}
+
+	[Mac (10,12, onlyOn64 : true)]
+	[BaseType (typeof(NSObject))]
+	interface WKOpenPanelParameters
+	{
```
Comment (dalexsoto) [inline src/wkwebkit.cs]: minor: brace should be one line above

**Comment by spouliot** [comment]
last build failed due to a checkout issue
```
hudson.plugins.git.GitException: Command "git submodule update --init --recursive external/mono" returned status code 143:
```

**Comment by spouliot** [comment]
build

**Comment by chamons** [comment]
Known test failure: https://bugzilla.xamarin.com/show_bug.cgi?id=55719

**Pull request merged**